pkg/metrics/metrics.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"time"
 
-	// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
+	// TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
```

Comment:
I've installed the latest golangci-lint (v1.44.0) by running `make lint` and it fails with the following message so I fixed it by removing the unnecessary `nolint`

```
pkg/metrics/metrics.go:9:2: directive `// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update` is unused for linter "gocritic" (nolintlint)
	// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
	^
make: *** [lint] Error 1
```

Reply:
How about adding this on #63 first?